Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi8/go-toolset:1.18.4-8.1669838000 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
```

Comment:
Can we please include the timestamp versionto make it more explicit if available